### ISSUE TITLE
hi，I installed 1.0.1+2, but something went wrong?

### ISSUE THREAD

**Issue by momtboy** [Mastersam07/livechat#8]
`[!] No podspec found for `livechatt` in `.symlinks/plugins/livechatt/ios``

I tried to install 1.0.1+1 and got the same result. Is this a scan code problem?

**Comment by momtboy** [comment]
`livechatt: ^1.0.1+1`
or
`livechatt: ^1.0.1+2`

**Comment by Mastersam07** [comment]
Hello @momtboy , can you try the latest version "**livechatt 1.0.1+3**"

**Comment by Mastersam07** [comment]
Let me know if it fixes your issue.

**Comment by momtboy** [comment]
@Mastersam07 hi,I have installed "**livechatt 1.0.1+3**", but it still does not work?

`livechatt: ^1.0.1+3`

`/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIWindowScene.h:17:50: note: 'UIWindowScene' has been marked as being introduced in iOS 13.0 here, but the deployment target is iOS 9.0.0
    UIKIT_EXTERN API_AVAILABLE(ios(13.0)) @interface UIWindowScene : UIScene
                                                     ^
    /Users/apple/work/workspace/code/flutter_app/piico_app/build/ios/Debug-iphonesimulator/LiveChat/LiveChat.framework/Headers/LiveChat-Swift.h:218:1: note: annotate 'setWindowScene:' with an availability attribute to silence this warning
    + (void)setWindowScene:(UIWindowScene * _Nullable)value;
    ^
    While building module 'LiveChat' imported from /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:3:
    In file included from <module-includes>:2:
    /Users/apple/work/workspace/code/flutter_app/piico_app/build/ios/Debug-iphonesimulator/LiveChat/livechat.framework/Headers/LiveChat-Swift.h:217:4: warning: 'UIWindowScene' is only available on iOS 13.0 or newer [-Wunguarded-availability-new]
    + (UIWindowScene * _Nullable)windowScene SWIFT_WARN_UNUSED_RESULT;
       ^
    In module 'UIKit' imported from /Users/apple/work/workspace/code/flutter_app/piico_app/ios/Pods/Target Support Files/LiveChat/LiveChat-umbrella.h:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIWindowScene.h:17:50: note: 'UIWindowScene' has been marked as being introduced in iOS 13.0 here, but the deployment target is iOS 9.0.0
    UIKIT_EXTERN API_AVAILABLE(ios(13.0)) @interface UIWindowScene : UIScene
                                                     ^
    While building module 'LiveChat' imported from /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:3:
    In file included from <module-includes>:2:
    /Users/apple/work/workspace/code/flutter_app/piico_app/build/ios/Debug-iphonesimulator/LiveChat/livechat.framework/Headers/LiveChat-Swift.h:217:1: note: annotate 'windowScene' with an availability attribute to silence this warning
    + (UIWindowScene * _Nullable)windowScene SWIFT_WARN_UNUSED_RESULT;
    ^
                                                                      API_AVAILABLE(ios(13.0))
    /Users/apple/work/workspace/code/flutter_app/piico_app/build/ios/Debug-iphonesimulator/LiveChat/livechat.framework/Headers/LiveChat-Swift.h:218:25: warning: 'UIWindowScene' is only available on iOS 13.0 or newer [-Wunguarded-availability-new]
    + (void)setWindowScene:(UIWindowScene * _Nullable)value;
                            ^
    In module 'UIKit' imported from /Users/apple/work/workspace/code/flutter_app/piico_app/ios/Pods/Target Support Files/LiveChat/LiveChat-umbrella.h:2:
    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator14.0.sdk/System/Library/Frameworks/UIKit.framework/Headers/UIWindowScene.h:17:50: note: 'UIWindowScene' has been marked as being introduced in iOS 13.0 here, but the deployment target is iOS 9.0.0
    UIKIT_EXTERN API_AVAILABLE(ios(13.0)) @interface UIWindowScene : UIScene
                                                     ^
    While building module 'LiveChat' imported from /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:3:
    In file included from <module-includes>:2:
    /Users/apple/work/workspace/code/flutter_app/piico_app/build/ios/Debug-iphonesimulator/LiveChat/livechat.framework/Headers/LiveChat-Swift.h:218:1: note: annotate 'setWindowScene:' with an availability attribute to silence this warning
    + (void)setWindowScene:(UIWindowScene * _Nullable)value;
    ^
                                                            API_AVAILABLE(ios(13.0))
    2 warnings generated.
    /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:13:4: error: unknown receiver 'SwiftLivechatPlugin'; did you mean 'LivechatPlugin'?
      [SwiftLivechatPlugin registerWithRegistrar:registrar];
       ^~~~~~~~~~~~~~~~~~~
       LivechatPlugin
    In file included from /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:1:
    /Users/apple/work/flutterSDK/flutter/.pub-cache/hosted/pub.flutter-io.cn/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.h:3:12: note: 'LivechatPlugin' declared here
    @interface LivechatPlugin : NSObject<FlutterPlugin>
               ^
    2 warnings and 1 error generated.`

**Comment by momtboy** [comment]
Hello，@Mastersam07 Is this bug fixed?

**Comment by akila89** [comment]
@Mastersam07, this issue not fixed in 1.0.1+3, I still receive following error. Is it pessary to upgrade to iOS13 can't we use iOS9? Please fix this asap or instruct me to how to fix this. :) 

    /Users/macair1/flutter/.pub-cache/hosted/pub.dartlang.org/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:13:4: error: unknown receiver 'SwiftLivechatPlugin'; did you mean 'LivechatPlugin'?
      [SwiftLivechatPlugin registerWithRegistrar:registrar];
       ^~~~~~~~~~~~~~~~~~~
       LivechatPlugin
    In file included from /Users/macair1/flutter/.pub-cache/hosted/pub.dartlang.org/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:1:
    /Users/macair1/flutter/.pub-cache/hosted/pub.dartlang.org/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.h:3:12: note: 'LivechatPlugin' declared here
    @interface LivechatPlugin : NSObject<FlutterPlugin>
               ^

**Comment by sandunangelo** [comment]
I'm also facing the same issue as @akila89 mentioned above, **is this issue have a workaround** until a proper fix is being released?

**Comment by EricBichara** [comment]
Also getting this error

**Comment by RClickDev** [comment]
> @Mastersam07, this issue not fixed in 1.0.1+3, I still receive following error. Is it necessary to upgrade to iOS13 can't we use iOS9? Please fix this asap or instruct me to how to fix this. :)
> 
> My flutter version
> Flutter 1.20.2 • channel stable • https://github.com/flutter/flutter.git
> Framework • revision bbfbf1770c (3 months ago) • 2020-08-13 08:33:09 -0700
> Engine • revision 9d5b21729f
> Tools • Dart 2.9.1
> 
> I can not update flutter version more than 1.20.4 at the moment
> 
> ```
> /Users/macair1/flutter/.pub-cache/hosted/pub.dartlang.org/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:13:4: error: unknown receiver 'SwiftLivechatPlugin'; did you mean 'LivechatPlugin'?
>   [SwiftLivechatPlugin registerWithRegistrar:registrar];
>    ^~~~~~~~~~~~~~~~~~~
>    LivechatPlugin
> In file included from /Users/macair1/flutter/.pub-cache/hosted/pub.dartlang.org/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.m:1:
> /Users/macair1/flutter/.pub-cache/hosted/pub.dartlang.org/livechatt-1.0.1+3/ios/Classes/LivechatPlugin.h:3:12: note: 'LivechatPlugin' declared here
> @interface LivechatPlugin : NSObject<FlutterPlugin>
>            ^
> ```

went from ios version error to this..looks like the wrong object was called. contacted the author on twitter. lets wait and see

**Comment by Mastersam07** [comment]
Hello, sorry. I've been busy for a while. I'll look into the issue and fix it. Will take the requirements down to iOS9 and fix from there.

**Comment by Mastersam07** [comment]
@RClickDev thanks for reaching out :+1:

**Comment by mustafa-707** [comment]
@Mastersam07 same here i got this : 

```
   While building module 'LiveChat' imported from
    /Users/admin/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/livechatt-1.0.1+3/ios/Classes/Live
    chatPlugin.m:3:
    In file included from <module-includes>:2:
    /Users/admin/Documents/myapp/app/build/ios/Release-iphoneos/LiveCha
    t/livechat.framework/Headers/LiveChat-Swift.h:497:1: note: annotate 'setWindowScene:' with an
    availability attribute to silence this warning
    + (void)setWindowScene:(UIWindowScene * _Nullable)value;
    ^
                                                            API_AVAILABLE(ios(13.0))
    2 warnings generated.
    /Users/admin/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/livechatt-1.0.1+3/ios/Classes/Live
    chatPlugin.m:13:4: error: unknown receiver 'SwiftLivechatPlugin'; did you mean 'LivechatPlugin'?
      [SwiftLivechatPlugin registerWithRegistrar:registrar];
       ^~~~~~~~~~~~~~~~~~~
       LivechatPlugin
    In file included from
    /Users/admin/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/livechatt-1.0.1+3/ios/Classes/Live
    chatPlugin.m:1:
    /Users/admin/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/livechatt-1.0.1+3/ios/Classes/Live
    chatPlugin.h:3:12: note: 'LivechatPlugin' declared here
    @interface LivechatPlugin : NSObject<FlutterPlugin>
               ^
    2 warnings and 1 error generated.
```

also when i tried to install v1.0.1+2 or v1.0.1+1 , it seems with same error with same name /livechatt-1.0.1+3/ can u checked it asap im also working to find a fix

**Comment by akila89** [comment]
@Mastersam07 Any fix on this issue?

**Comment by JediIzzyT** [comment]
@Mastersam07 Just bumping. Everything is dandy on Android, but my iOS app crashes because of this issue. I do have some additional information that may help fix it though. I had the same error as @momtboy originally. I removed the livechatt plugin from my app, flutter cleaned, then rebuilt. My app was working without livechatt just fine, so I thought I would try to fix it again, but after I added livechatt back in, I got this error when building in Xcode: 
`Compiling for iOS 9.0, but module 'LiveChat' has a minimum deployment target of iOS 11.0: /Users/{omitted}/Library/Developer/Xcode/DerivedData/Runner-affukayupdwoauadreqpwvhdnvjw/Build/Products/Debug-iphonesimulator/LiveChat/LiveChat.framework/Modules/LiveChat.swiftmodule/x86_64-apple-ios-simulator.swiftmodule`

The error references line 3 of `SwiftLivechatPlugin.swift`

This makes me think that the library you are referencing with the iOS side of the plugin has a minimum target of iOS 11, but the livechatt plugin itself targets iOS 9, making the two incompatible and causing errors. Could you change the plugin's target iOS version in a future release?

**Comment by Mastersam07** [comment]
@JediIzzyT  Thanks for this update. Will look into it. I was planning rolling back to LiveChat: 0.0.4 and fixing from there but this helps alot.

**Comment by JediIzzyT** [comment]
@Mastersam07 When personally messing with the package, I was able to get past the compilation error by changing the target OS from iOS 9.0 to iOS 11.0 in `ios\livechatt.podspec` and then running `pod install` in the directory. However, a new issue presents itself after that, and although it is unique to this package within my own project, it is not a unique issue overall. However, I am relatively new to iOS development, so I submitted a question about it here instead of continuing to struggle on my own: [https://stackoverflow.com/questions/66228140/flutter-plugin-throwing-error-thread-1-exc-bad-access-on-ios](https://stackoverflow.com/questions/66228140/flutter-plugin-throwing-error-thread-1-exc-bad-access-on-ios)

**Comment by JediIzzyT** [comment]
Hello @Mastersam07 
I noticed that you closed this issue. Do you plan on releasing an update that solves this? It is still not working for me. Thanks!

**Comment by Mastersam07** [comment]
Yes. I am already publishing an update. Fixed and tested on iOS.
![Simulator Screen Shot - iPhone 11 Pro Max - 2021-04-22 at 11 49 34](https://user-images.githubusercontent.com/31275429/115769825-d8895980-a360-11eb-9114-874943998f52.png)

**Comment by Mastersam07** [comment]
I will leave this open until i merged the PR to master. You can follow from this PR #12

**Comment by sandunangelo** [comment]
> Pr [merged](https://github.com/Mastersam07/livechat/runs/2413430654). version 1.1.0 is live. @JediIzzyT @akila89 @sandunangelo @RClickDev this should fix your issues.
> 
> @CheapSmokes this should also be a final fix to #6

Thanks @Mastersam07  now it works fine.
